README.md

```diff
@@ -238,11 +238,10 @@ When developing a new component you should generally follow three rules:
 
 ##### 1. Start the documentation site development server
 
-The Paragon documentation site serves both as documentation and as a workbench to create your component within. To see your component in action, you need to run the documentation site locally. (Note you need to install dependencies both in the project root and the `www` directory)
+The Paragon documentation site serves both as documentation and as a workbench to create your component within. To see your component in action, you need to run the documentation site locally.
 
 ```
 npm install
-cd www
```

Comment:
Love that we no longer need to `cd www` in order to start the docs site locally now :)

Reply:
Yeah, that was definiteley annoying sometimes haha